padertorch/contrib/examples/source_separation/or_pit/train.py

```diff
@@ -163,7 +173,8 @@ def prepare_iterable(
     if iterator_slice is not None:
```

Comment:
Can you use a normal function instead of `lambda x: (x.update(num_speakers=len(x['speaker_id'])), x)[1]`?

Reply:
Done